app/Http/Services/EventService.php

```diff
@@ -56,15 +56,24 @@ public function update(EventRequest $request, $id)
 	 * Cancel an event.
 	 * @param int $id event id
 	 */
-	public function destroy($id, $organization)
+	public function cancel($id)
 	{
 		$event = Event::findOrFail($id);
-		if($organization->id == $event->organization()->id)
-		{
-			$event->delete();
-			Notification::notify($event->volunteers, 3, null,
-								"Event ".($event->name)."has been cancelled", url("/"));
-		}
+		$event->delete();
+		Notification::notify($event->volunteers, 3, null,
+			"Event ".($event->name)."has been cancelled", url("/"));
+		$this->unindexEvent($event->id);
+	}
+
+	/**
+	 * Delete an event.
+	 */
+	public function destroy($id)
+	{
+		$event = Event::findOrFail($id);
+		$event->forceDelete();
+		Notification::notify($event->volunteers, 3, null,
```

Comment:
maybe i don't need to unindex but i need to notify and that was stated in the issue